apps/src/applab/applab.js

```diff
@@ -62,6 +59,9 @@ import {
   actions as jsDebugger,
 } from '../lib/tools/jsdebugger/redux';
 import JavaScriptModeErrorHandler from '../JavaScriptModeErrorHandler';
+import connectToMakerBoard from '../lib/kits/maker/connectToMakerBoard';
+import * as makerCommands from '../lib/kits/maker/commands';
+import * as makerDropletConfig from '../lib/kits/maker/dropletConfig';
```

Comment:
At one point I gave maker an index.js so I could import like this:
```javascript
import {
  connectToMakerBoard,
  commands as makerCommands,
  dropletConfig as makerDropletConfig
} from '../lib/kits/maker';
```

I kind of like it, but I think at some point the team decided we don't like the index.js convention?